src/cron/index.ts

```diff
@@ -42,3 +31,13 @@ export function setupCronJobs(app: App): () => void {
     scheduledJobs.forEach(scheduledJob => scheduledJob());
   };
 }
+
+function getJobs(): ScheduledJob[] {
+  const everyFifteenWorkDay = `*/15 ${process.env.WORKDAY_START_HOUR}-${process.env.WORKDAY_END_HOUR} * * MON-FRI`;
```

Comment:
Moved this to a method because the `const` up top was evaluating prior to `process.env` being loaded. By moving it to this method the start and end hours should be populated from the environment variables.